### ISSUE TITLE
How can i get already configured environment?

### ISSUE THREAD

**Issue by hiteshbalraw** [mozilla/nunjucks#639]
I want the way of getting the previously configured environment in nunjucks, rather than reconfiguring.

**Comment by carljm** [comment]
There's no API currently available for this. Can't you just keep a reference to the environment when it's returned from the initial `configure` call?

I guess I wouldn't be opposed to a PR that exposes a new API for getting the last-configured env.

**Comment by dandart** [comment]
I agree, I;d like to be able to do:

require nunjucks
configure env


later on in another file
require nunjucks
nunjucks.render(something)

and have the filters configured earlier execute.

**Comment by carljm** [comment]
@dandart I don't agree that that's a desirable pattern (too much implicit state), but it is how nunjucks currently works, and it should work right now. You don't need the change proposed in this ticket for your pseudo code; nowhere in it do you reach back in in the second file to grab a reference to the previously configured env.

**Comment by dandart** [comment]
At the moment I'm trying to find a way to import my wrapper library and have it run on all instances.
